### PR TITLE
Add stopping condition to MainLoop

### PR DESCRIPTION
**Pull request by moraval**

### CODE REVIEW

File: emloop/main_loop.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def __init__(self,   # pylint: disable=too-many-arguments
                  train_stream_name: str=EL_DEFAULT_TRAIN_STREAM,
                  extra_streams: Iterable[str]=(),  # pylint: disable=invalid-sequence-index
                  buffer: int=0,
+                 epochs_count: int=1,
```
Comment (blazekadam) [inline emloop/main_loop.py]:
```suggestion
                 epochs_count: Optional[int]=None,
```
lets make this `None` by default and do not stop if its not specified

**Review by blazekadam — CHANGES_REQUESTED** [review]
Please implement some tests for this.

**Comment by blazekadam** [comment]
I still miss the tests...

**Comment by moraval** [comment]
I didn't see the comment.. Will do.

**Review by blazekadam — APPROVED** [review]

**Pull request merged**